aptos-move/aptos-vm/src/move_vm_ext/session.rs

```diff
@@ -191,14 +191,17 @@ impl<'r, 'l> SessionExt<'r, 'l> {
     ) -> VMResult<(MoveChangeSet, MoveChangeSet)> {
         // The use of this implies that we could theoretically call unwrap with no consequences,
         // but using unwrap means the code panics if someone can come up with an attack.
-        let common_error = PartialVMError::new(StatusCode::UNKNOWN_INVARIANT_VIOLATION_ERROR)
-            .with_message("split_and_merge_resource_groups error".to_string())
-            .finish(Location::Undefined);
+        let common_error = || {
```

Comment:
lambda ftw, i think I've seen the prior clone pattern often and I like this strictly better